pkg/utils/user/user.go

```diff
@@ -0,0 +1,22 @@
+package user
+
+import (
+	"context"
+
+	userv1 "github.com/openshift/api/user/v1"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// GetUsersByOwnerName gets the user resources by matching owner label.
+func GetUsersByOwnerName(cl client.Client, owner string) ([]userv1.User, error) {
+	userList := &userv1.UserList{}
```

Comment:
Could you add some tests for this utility func?

Reply:
Actually, since it was only used by the useraccount controller I moved it to a local function there and the test coverage is good as well. Done in 55da18804f2cf11841825cb7a3866bfc9813813d

Reply:
ok, thanks